tests/Unit/Subscriber/Behavior/Path/AutoNameSubscriberTest.php

```diff
@@ -171,6 +173,7 @@ public function testAutoName()
     public function testAutoNameWithDisabledOption()
     {
         $this->persistEvent->getOption('auto_name')->willReturn(false);
+        $this->persistEvent->getDocument()->willReturn(null);
```

Comment:
Shouldn't a document of type `AutonameBehavior` be returned here? Otherwise the test would also succeed if  `auto_name` is not set to false.